pages/services/data-science-engine/1.0.0/ssl/index.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: layout.pug
+navigationTitle: SSL
+excerpt: SSL/TLS Support 
+title: SSL
+menuWeight: 8
+model: /services/data-science-engine/data.yml
+render: mustache
+---
+
+# {{ model.techName }} SSL
```

Comment:
Can we make this a child page? Similar to how [DCOS enterprise security](https://docs.mesosphere.com/1.13/security/ent/) page has 
```
DC/OS Enterprise Security
- Managing Users and Groups
- Granting Access to the UI
...
```
It feels like SSL falls under Security. Rather than its own page. WDYT?

Reply:
I know SSL is more common but the service config has all the settings under `tls`. Should we rename this file to TLS?

Reply:
@samvantran, we've discussed with docs team and agreed on having TLS page as a separate one to keep it similar to Kerberos config